Tests/govuk_ios/govuk_ios_snapshot_tests/Helpers/SnapshotTestCase.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+import iOSSnapshotTestCase
+
+@testable import govuk_ios
+
+class SnapshotTestCase: FBSnapshotTestCase {
+
+    override func setUp() {
+        super.setUp()
+//        self.recordMode = true
```

Comment:
A comment appears to be left in by accident

Reply:
This is deliberate. Its to uncomment when/ if we want to retake all snapshot tests.